src/events/keyboard.ts

```diff
@@ -98,9 +98,12 @@ function handleTabKey(scrollbar: I.Scrollbar) {
 }
 
 function isEditable(elem: any): boolean {
-  if (elem.tagName === 'INPUT' || elem.tagName === 'TEXTAREA') {
+ if (
+    elem.tagName === "INPUT" ||
+    elem.tagName === "TEXTAREA" ||
+    elem.contentEditable === "true"
```

Comment:
The problem is that the value of `contentEditable` attribute may be `inherit` (default value), which means you have to traverse through all the ancestor elements to see if this one is editable.

Reply:
A simple but high-cost way to fix it:

```js
while (elem) {
  if (elem.contentEditable === "true") {
     // do something
    break;
  }

  elem = elem.parentElement;
}
```

Reply:
Just got an answer from [stack overflow](https://stackoverflow.com/a/25760172/2369823):

```diff
- elem.contentEditable === "true"
+ elem.isContentEditable
```

@Alecyrus can you modify your code like the above snippet?